ci/image-skipping-logger.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This helper script is used by notebooks-digest-updater.sh and runtimes-digest-updater.sh 
```

Comment:
Hmm, I think that it's used only in the `sha-digest-updater.sh` so we don't need to have this defined as a separate script anymore?

Reply:
Great catch! I removed that